pkg/metadata/metadata.go

```diff
@@ -34,7 +34,7 @@ type DatabasesMeta struct {
 }
 
 type TableMetadata struct {
-	Files map[string][]string `json:"files,omitempty"`
+	DiskToFiles map[string][]string `json:"files,omitempty"`
```

Comment:
Everything looks good, 
thanks a lot for contributing, but please let's stay `Files` here
I really care about mapping between JSON structure and variable names